docs/doc_v1.0/Build_documentation.md

```diff
@@ -80,10 +91,18 @@ from outside the network interacting. This is the probability per
 person, per timestep. So, 0.05 is 5 people in a population of 100 per
 day. This value is likely to be much smaller than 0.05.
 
-![](image3.png)
+<img src="image3.png" alt="Contacts File" width="400">
+
 
 Figure 3. The diseaseSettings.json file
 
+
+The exposureProbability4UnitContact (expUnitContact) and exposureExponent (exp) values are 
+used to determine the chance of infection for a given contact. 
```

Comment:
I think the mathematical implementation may be worth explicitly declaring.
Also the name _exposureProbability4UnitContact_ sounds too similar to the Java unit testing module. Perhaps a more descriptive name should be added at some point.